dev-libs/boehm_gc/boehm_gc-7.6.4.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="The Boehm-Demers-Weiser conservative garbage collector"
+DESCRIPTION="The Boehm-Demers-Weiser conservative garbage collector can be \
+used as a garbage collecting replacement for C malloc or C++ new. It allows \
+you to allocate memory basically as you normally would, without explicitly \
+deallocating memory that is no longer useful. The collector automatically \
+recycles memory when it determines that it can no longer be otherwise \
+accessed."
+HOMEPAGE="http://www.hboehm.info/gc/"
+COPYRIGHT="1988, 1989 Hans-J. Boehm, Alan J. Demers
+	1991-1996 Xerox Corporation
+	1996-1999 Silicon Graphics
+	1999-2011 Hewlett-Packard Development Company, L.P."
+LICENSE="Boehm"
+REVISION="1"
+SOURCE_URI="https://github.com/ivmai/bdwgc/releases/download/v$portVersion/gc-$portVersion.tar.gz"
+CHECKSUM_SHA256="b94c1f2535f98354811ee644dccab6e84a0cf73e477ca03fb5a3758fb1fecd1c"
+SOURCE_DIR="gc-$portVersion"
+PATCHES="boehm_gc-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 ?x86"
+
+libcordVersion="1.3.0"
+libgcVersion="1.3.2"
+libcordVersionCompat="$libcordVersion compat >= ${libcordVersion%%.*}"
+libgcVersionCompat="$libgcVersion compat >= ${libgcVersion%%.*}"
```

Comment:
It's prefered that one uses camelCase for local variables, but it's not strictly required.

```diff
-libcordVersion="1.3.0"
-libgcVersion="1.3.2"
-libcordVersionCompat="$libcordVersion compat >= ${libcordVersion%%.*}"
-libgcVersionCompat="$libgcVersion compat >= ${libgcVersion%%.*}"
+libVersion="1.3.2"
+libCordVersion="1.3.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+libCordVersionCompat="$libCordVersion compat >= ${libCordVersion%%.*}"

Reply:
Well, if we consider that libcord and libgc are words, then we already have a sort of camelCase :)

Reply:
BTW, keeping the basename of the libraries is nice because with recipes that have many libraries we can use the eval trick I recently used for ffmpeg.
And the wiki does not mention camelCase. `libVersion` and ``libVersionCompat` happen to be camelCase, but when we have more than just one library, then I think it is nice to use the original basename :)